install/scripts/aws-down.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Required env variables:
+: "$AWS_ACCOUNT_ID $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+kubernetes_dir=${script_dir}/../kubernetes
+
+EKSCTL_ENABLE_CREDENTIAL_CACHE=1
+export CLUSTER_NAME=substratus
+export REGION=us-west-2
+export ARTIFACTS_REPO_NAME=${CLUSTER_NAME}
+export ARTIFACTS_BUCKET_NAME=${AWS_ACCOUNT_ID}-${CLUSTER_NAME}-artifacts
+
+(aws eks update-kubeconfig \
+  --region ${REGION} \
+  --name ${CLUSTER_NAME} &&
+  kubectl delete deployments --namespace=karpenter --all &&
```

Comment:
Any reason to delete these namespaces as opposed to blasting the cluster?

Reply:
Weirdly, the cloudformation stack will find itself stuck and unable to actually delete the node groups due to unevictable pods in these namespaces. See: https://github.com/substratusai/substratus/pull/170#discussion_r1289325977 for why not just delete the cluster.

I suppose one approach might be deleting the namespaces. That would probably be a suitable alternative, esp if other resource types are blocking (daemonsets)

Reply:
Thanks for the context. Can you add a comment to describe the need so I don’t forget and try to “simplify” later?